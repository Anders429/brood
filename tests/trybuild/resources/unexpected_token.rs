use brood::resources;

// Define resources.
struct A;
struct B;

fn main() {
    let resources = resources!(A, + B);
}
