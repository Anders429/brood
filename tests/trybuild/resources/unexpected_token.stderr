error: no rules expected the token `+`
 --> tests/trybuild/resources/unexpected_token.rs:8:35
  |
8 |     let resources = resources!(A, + B);
  |                                   ^ no rules expected this token in macro call
  |
note: while trying to match meta-variable `$resources:expr`
 --> src/resource/mod.rs
  |
  |     ($resource:expr $(,$resources:expr)* $(,)?) => {
  |                        ^^^^^^^^^^^^^^^
